pkg/adapter/awscodecommitsource/adapter.go

```diff
@@ -87,27 +82,18 @@ func NewAdapter(ctx context.Context, envAcc pkgadapter.EnvConfigAccessor,
 	env := envAcc.(*envConfig)
 
 	// create CodeCommit client
-	sess, err := session.NewSession(&aws.Config{
-		Region:      &env.AWSRegion,
-		Credentials: credentials.NewStaticCredentials(env.AccountAccessKeyId, env.AccountSecretAccessKey, ""),
-		MaxRetries:  aws.Int(5),
-	})
-	if err != nil {
-		logger.Fatalw("Failed to create CodeCommit client", "error", err)
-	}
+	sess := session.Must(session.NewSession(aws.NewConfig().WithMaxRetries(5)))
```

Comment:
Do we need to specify the region with `aws.NewConfig()`?

```suggestion
	sess := session.Must(session.NewSession(aws.NewConfig().WithRegion(env.AWSRegion).WithMaxRetries(5)))
```

Reply:
The region is also set from the environment (`AWS_REGION`). If this env var is missing, the client falls back to reading your local config, which is quite handy when debugging sources locally. (e.g. I can have it default to eu-central and you to us-west without any explicit parameter for example)

Reply:
thanks for the clarification